src/services/telemetryService.ts

```diff
@@ -82,7 +88,7 @@ const analyticsClientFactory = async (key: string, logger: vscode.LogOutputChann
 };
 
 interface TelemetryOptions {
-	gitpodHost?: string;
+	gitpodHost: string;
```

Comment:
wouldn't it be possible to add `gitpodHost` automatically  in the `${gitpodHost}/analytics`  endpoint as the body is just json, this will work for all events not just the ones required explicitly by UserFlowTelemetry and we wouldn't have to worry about forgetting sending it

Reply:
I did not get it, sorry. Could you elaborate what changes are necessary?

Reply:
[Here](https://github.com/gitpod-io/gitpod/blob/d5525e3dc938428af34c2dd187986e1fb4c35d8f/components/proxy/plugins/analytics/analytics.go#L72-L90) intercepet the request, parse the json body and add `gitpodHost` property instead of doing it for every event in each client

Reply:
Ah, for property, we should completely remove it. It is not necessary anymore.